Guideline for DLT Multinode demo

duvanan13 opened the pull request:
Guideline for DLT Multinode demo:
+ Setup Docker
+ Setup network bridge
+ Fetch logs from Container to Host

minminlittleshrimp commented on Section_4_Demo_docker/test.md:
```diff
@@ -0,0 +1,4 @@
+# Hướng Dẫn Multinode Demo với Docker Setup
```
Please prepare the doc in English

minminlittleshrimp commented on Section_4_Demo_docker/test.md:
```diff
@@ -0,0 +1,4 @@
+# Hướng Dẫn Multinode Demo với Docker Setup
+
+ Multinode Demo với Docker Setup tại [đây](https://inside-docupedia.bosch.com/confluence2/display/EASHome/Multinode+Demo+with+Docker+setup).
```
Why attaching an internal link here?
@duvanan13  please prepare your own doc to fit in with OSS community.

duvanan13 replied:
I'm sorry for being so subjective so I have update new changes below.
Thanks for remind me!!!
<3

minminlittleshrimp commented on Section_4_Demo_docker/test.md:
````diff
@@ -0,0 +1,132 @@
+## 2.7. Install dlt bin and lib in the container:
+
+```bash
+root@2ff461cda2f5:/home/nad8hc/work/docker/dlt-daemon# cp ./build/src/daemon/dlt-daemon /usr/local/bin/
````
replace nad8hc with <$USER>

minminlittleshrimp committed:
dlt: demo multinode

Guideline for DLT Multinode demo:

Setup Docker
Setup network bridge
Fetch logs from Container to Host

Signed-off-by: LUU QUANG MINH <Minh.LuuQuang@vn.bosch.com>
